tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -3019,9 +3022,9 @@ def refit_full(self, model="all", set_best_to_refit_full=True):
                 model_to_set_best = set_best_to_refit_full
             else:
                 model_to_set_best = model_best
-            model_full_dict = self._trainer.get_model_full_dict()
-            if model_to_set_best in model_full_dict:
-                self._trainer.model_best = model_full_dict[model_to_set_best]
+            model_refit_map = self._trainer.model_full_dict()
```

Comment:
I've now switched to using the consistent `model_refit_map` name everywhere. I originally didn't do this because TimeSeries also used `model_full_dict` and I was avoiding changing TimeSeries, but I have made the refactor for TimeSeries as well now.